Resources/Prototypes/Entities/Objects/Devices/door_remote.yml

```
@@ -37,6 +37,34 @@
       enum.DoorRemoteUiKey.Key:
         type: DoorRemoteBoundUserInterface
 
+- type: entity
+  parent: DoorRemoteDefault
+  id: DoorRemoteCanEletrifyDoors
+  abstract: true
+  components:
+  - type: DoorRemote
+    options:
+    - mode: ToggleOvercharge
+      tooltip: door-remote-toggle-eletrify-text
+      icon:
+        sprite: /Textures/Interface/Actions/actions_ai.rsi
+        state: door_overcharge_on
+    - mode: OpenClose
+      tooltip: door-remote-open-close-text
+      icon:
+        sprite: /Textures/Structures/Doors/Airlocks/Standard/basic.rsi
+        state: assembly
```

Place anchors on the parent so that you can alias these.
```suggestion
      tooltip: *TextOpenClose
      icon: *IconOpenClose
```